.github/actions/create-timestamp/action.yml

```diff
@@ -0,0 +1,16 @@
+﻿name: create-timestamp
+author: qwerty
+description: Create a timestamp in ISO 8601 format for use in tags (replaces ":" with "." for use in git tags).
+runs:
+  using: composite
+  steps:
+    - name: Get timestamp
```

Comment:
Use American Style Titles in all of these step names, in the other files too.

Reply:
Not addressed.

Reply:
It was addressed here no? https://github.com/Lombiq/GitHub-Actions/blob/c89918130cb5f7867b4190bea8f4db1daaabb990/.github/actions/create-timestamp/action.yml#L6

Reply:
I asked for all steps in all affected files to be named like this.